app/packages/app/src/components/QueryPerformanceToast.tsx

```diff
@@ -1,13 +1,14 @@
 import { Toast } from "@fiftyone/components";
 import { QP_MODE } from "@fiftyone/core";
 import { getBrowserStorageEffectForKey } from "@fiftyone/state";
-import { Box, Button, Typography } from "@material-ui/core";
+import { Box, Button, Typography } from "@mui/material";
 import { Bolt } from "@mui/icons-material";
 import React, { useEffect, useState } from "react";
 import { createPortal } from "react-dom";
 import { atom, useRecoilState } from "recoil";
+import { useTheme } from "@fiftyone/components";
 
-const SHOWN_FOR = 10000;
+const SHOWN_FOR = 50000;
```

Comment:
This might be 50 seconds instead of 5 seconds btw ~

Reply:
I just copied this from your PR: https://github.com/voxel51/fiftyone/pull/4966. Do you wanna merge that one and I can rebase onto it or merge this one and do the opposite?

Reply:
My bad, my PR was WIP so I set a high number for debugging; you can set to `5000` and merge directly.